cypress/e2e/flow-builder/connections/creating-removing.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Creating a connection', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'Creating a connection',
+    })
+
+    cy.addBlock(['Content', 'Message']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('firstBlock')
+    })
+
+    cy.addBlock(['Content', 'Numeric Response']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('secondBlock')
+    })
+
+    // wait for flow validation after adding blocks
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(300)
+  })
+
+  it('creates and removes a connection', () => {
+    cy.get('@firstBlock').find('[data-cy^="exit--"]').as('exit')
```

Comment:
1st time I see this ^ modifier, nice!

Reply:
cc: @object-required , @pascaleli in case you too

Reply:
I think I saw it in @object-required's PR :) 
https://github.com/FLOIP/flow-builder/pull/481/files#diff-6c107e2390c5ad84e123f3be09d7b91ba35de8f8cc550d1f358c75806df47645R100